common/io/EPoller.cpp

```diff
@@ -312,7 +312,7 @@ bool EPoller::Poll(TimeoutManager *timeout_manager,
 
   int ms_to_sleep = sleep_interval.InMilliSeconds();
   int ready = epoll_wait(m_epoll_fd, reinterpret_cast<epoll_event*>(&events),
-                         MAX_EVENTS, ms_to_sleep ? ms_to_sleep : 1);
+                         MAX_EVENTS, ms_to_sleep ? ms_to_sleep : 0);
```

Comment:
Could we have just this change cherry-picked into a new PR targeting 0.10 branch please @DaAwesomeP ?